NOTICE.txt

```diff
@@ -7,20 +7,59 @@ The Apache Software Foundation (http://www.apache.org/).
 Based on source code originally developed by
 Baidu (http://www.baidu.com/).
 
-This product includes software developed by the LevelDB.
-Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+--------------------------------------------------------------------------------
 
-This product includes software developed by CPython for searching string function.
-(https://github.com/python/cpython)
+This product includes code from Apache commons-lang, which includes the following in
+its NOTICE file:
 
-This product includes software developed by Chromium.
-Copyright (c) 2011 The Chromium Authors. All rights reserved.
+  Apache commons-lang
+  Copyright 2001-2020 The Apache Software Foundation
 
-This product includes software developed by Austin Appleby for MurmurHash3.
-The author hereby disclaims copyright to this source code.
+  This product includes software developed at
+  The Apache Software Foundation (http://www.apache.org/).
```

Comment:
I think there is no need to put `Apache commons-lang`' NOTICE file here, because it is a Apache software and there is no other special statement in its NOTICE.